api/templates/registration/password_change_form.html

```diff
@@ -10,9 +10,22 @@
 </nav>
 
   <h2>Change password</h2>
-  <form method="post">
-    {% csrf_token %}
-    {{ form.as_p }}
-    <button type="submit">Change password</button>
+  <script src="//cdnjs.cloudflare.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
```

Comment:
how come ajax include is added here? Should be in `base.html` template already?

Reply:
Yeah I think you're right theres no need for it to be here so I'll just remove it